packages/auth/src/types.ts

```diff
@@ -8,3 +8,8 @@ export type MadAuthenticationResult = {
     account: MadAccount;
     accessToken: string;
 };
+export type UserInfo = {
```

Comment:
Do we want to leak out this type to consumers of the package?
If this type is only needed to get typings on the parsing of the jwt-token, maybe it is better left closer to the decoding method.
I suggest maybe moving the `decodeToken` and this type to a separate file. This makes it much easier to write tests for as well (not that the decode method needs it unless we add more business logic to it)

Reply:
created `decodeToken.ts` and moved UserInfo to the file as a private type